.gitignore

```diff
@@ -0,0 +1,21 @@
+ckup files from text editors
+*~
+*.DS_Store
+.*.swp
+.hg
+.hgignore
+.svn
+.svnignore
+
+# Xcode crap
+*mode1v3
+*pbxuser
+*perspectivev3
+*xcuserdata*
+# Junk files on Mac OS X
+*.DS_Store
+
+#build
+#include
+docs
+
```

Comment:
please, delete gitignore changes. In local gitignore you should story only project-specific ignore rules. Everything else should be placed in global gitignore.